.taprc

```diff
@@ -0,0 +1 @@
+coverage: false
```

Comment:
Is it better to turn on coverage and disable check-coverage only? The other three is added for consistency through all repository.

```suggestion
ts: false
jsx: false
flow: false
check-coverage: false
coverage: true
```